contrib/fuzzing/build.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 cd "${0%/*}"
 cd ../..
-AFL_USE_ASAN=1 CONFIG=afl make clean all EXAMPLES=0 TEST=0 OPENMPT123=0 NO_VORBIS=1 NO_VORBISFILE=1 NO_MPG123=1 CHECKED_ADDRESS=1
+AFL_LLVM_CMPLOG=1 AFL_USE_ASAN=1 CONFIG=afl make clean all EXAMPLES=0 TEST=0 OPENMPT123=0 NO_VORBIS=1 NO_VORBISFILE=1 NO_MPG123=1 CHECKED_ADDRESS=1
```

Comment:
Doesn't this addition require building the binary two times (one with cmplog, one without)? Is there any benefit in building just one binary with cmplog?

Reply:
Yes that would be beneficial. This is the easier way. As you use an asan binary anyway it doesn’t matter. Best practice would be 3 binaries :) normal, cmplog, asan